scripts/app.js

```diff
@@ -1,44 +1,43 @@
 (function(module) {
 
-function Project(opts) {
-  'use strict';
-  this.author = opts.author;
-  this.title = opts.title;
-  this.category = opts.category;
-  this.authorUrl = opts.authorUrl;
-  this.publishedOn = opts.publishedOn;
-  this.body = opts.body;
-}
+  function Project(opts) {
+    this.author = opts.author;
+    this.title = opts.title;
+    this.category = opts.category;
+    this.authorUrl = opts.authorUrl;
+    this.publishedOn = opts.publishedOn;
+    this.body = opts.body;
+  }
 
-Project.all = [];
+  Project.all = [];
 
-Project.prototype.toHtml = function() {
-  var template = Handlebars.compile($('#project-template').text());
-  return template(this);
-};
+  Project.prototype.toHtml = function() {
+    var template = Handlebars.compile($('#project-template').text());
+    return template(this);
+  };
 
-Project.loadAll = function(rawData) {
-  rawData.sort(function(a,b) {
-    return (new Date(b.publishedOn)) - (new Date(a.publishedOn));
-  });
-  rawData.forEach(function(ele) {
-    Project.all.push(new Project(ele));
-  });
-};
+  Project.loadAll = function(rawData) {
+    rawData.sort(function(a,b) {
+      return (new Date(b.publishedOn)) - (new Date(a.publishedOn));
+    });
+    rawData.forEach(function(ele) {
+      Project.all.push(new Project(ele));
```

Comment:
Same as what Jeff said, this can be optimized a little better with .map() (Functional Programming!).